MomentumDiscordBot/Commands/Admin/AdminConfigModule.cs

```diff
@@ -6,21 +6,22 @@
 using DSharpPlus;
 using DSharpPlus.CommandsNext;
 using DSharpPlus.CommandsNext.Attributes;
+using DSharpPlus.SlashCommands;
+using DSharpPlus.SlashCommands.Attributes;
 using DSharpPlus.Entities;
 using MomentumDiscordBot.Constants;
 using MomentumDiscordBot.Models;
 using HiddenAttribute = MomentumDiscordBot.Models.HiddenAttribute;
 
 namespace MomentumDiscordBot.Commands.Admin
 {
-    [Group("config")]
+    [SlashCommandGroup("config","show and modify the bot config")]
```

Comment:
This is very nitpicky, but in many places you're missing spaces after commas. I would suggest using an auto formatter as they'll handle cases like this easily, as otherwise it'll be tedious for you to fix everything manually.